css/projects_table.css

```diff
@@ -0,0 +1,37 @@
+* {
+    box-sizing: border-box;
+}
+.table table {
+    color: white;
+    border: none;
```

Comment:
Not sure but maybe one of the border properties is not necessary, could you please check? :)